lib/utils.js

```diff
@@ -134,40 +134,92 @@ module.exports = {
   },
 
   /**
+   *
+   * References:
+   * - https://github.com/facebook/react-native/blob/main/Libraries/Utilities/Platform.ios.js
+   * - https://github.com/facebook/react-native/blob/main/Libraries/Utilities/NativePlatformConstantsAndroid.js
+   *
    * @returns An object with names and versions of the various components making up the context.
    */
   getVersions() {
     const packageJson = require("../package.json");
-    const packageVersion = packageJson.version;
+    const sdkVersion = packageJson.version;
     const environment = this.getEnvironment();
 
+    const sdk = "JS";
+
     try {
       if (environment === "reactnative") {
         const { Platform } = require("react-native");
+        let deviceName = "unknown";
+        let deviceVersion = "unknown";
+        if (Platform.OS === "ios") {
+          if (Platform.isPad()) {
+            deviceName = "iPad";
+          } else if (Platform.isTV()) {
```

Comment:
Hey @kneth, these caused a regression since `Platform.isPad` and `Platform.isTV` are boolean properties (not methods).

```
Error getting versions: TypeError: false is not a function
```

Please see: https://reactnative.dev/docs/platform#ispad-ios

Reply:
@appden Thank you for reporting. I have a fix in https://github.com/realm/realm-js/pull/5491.